packages/authenticator/amplify_authenticator/example/macos/Runner/Release.entitlements

```diff
@@ -6,5 +6,9 @@
 	<true/>
 	<key>com.apple.security.network.client</key>
 	<true/>
+	<key>keychain-access-groups</key>
```

Comment:
Can we align with https://github.com/aws-amplify/amplify-flutter/pull/2396?

I found this to be a nicer experience than requiring a development account right off the bat.

Reply:
Yeah. I agree that is probably better. You will occasionally have to allow access, which is unfortunately done on a key by key basis, so you have to allow access ~11 times by entering your password. Still probably better than the alternative for most people.